chaosk8s/statefulset/actions.py

```diff
@@ -1,4 +1,7 @@
 # -*- coding: utf-8 -*-
+import json
+import os.path
+import yaml
```

Comment:
that import should be after the kubernetes one as it's a third-party

Reply:
I should clarify, the `yaml` import is a third-party so its place should be after the last third-party imports in the next block.

We tend to group imports as follows:

stdlib

third-party

local

:)

Reply:
From other spots, third-party imports are above kubernetes (with a blank line separation)
https://github.com/chaostoolkit/chaostoolkit-kubernetes/blob/master/chaosk8s/pod/actions.py

Reply:
Yeap, they are wrong and should be fixed. Let's ensure yours is good from the get go :)

Reply:
Ok, so I can be sure 
Like this ?

> from chaoslib.exceptions import ActivityFailed
> from chaoslib.types import Secrets
> from kubernetes import client
> from kubernetes.client.rest import ApiException
> from logzero import logger
> 
> import json
> import os.path
> import yaml

Reply:
I searched a bit more for example and i change for this : 

> import json
> import os.path
> 
> from chaoslib.exceptions import ActivityFailed
> from chaoslib.types import Secrets
> from kubernetes import client
> from kubernetes.client.rest import ApiException
> from logzero import logger
> import yaml

json and os.path are stdlib, right ?
(Sorry I'm newbie in python)

Reply:
Yes. That's the right one. 

Indeed, json and os.path are stdlib :)